katcp/test/test_fake_clients.py

```diff
@@ -35,6 +35,12 @@ def test_sensors(self):
         sensor_info = {
             'an-int': ('An integer sensor', 'things', 'integer', 0, 10),
             'a-string' : ('A string sensor', '', 'string'),
+            'a-discrete': ('A discrete sensor', '', 'discrete', 'a', 'b', 'c'),
+            'a-timestamp': ('A timestamp sensor', '', 'timestamp', 1556928000),
+            'a-float': ('A float sensor', '', 'float', 1234.42),
+            'a-boolean': ('A boolean sensor', '', 'boolean', '0'),
+            'an-address': ('An address sensor', '', 'address', '127.0.0.1:8080'),
+            'a-lru': ('A LRU sensor', '', 'lru', 'nominal'),
```

Comment:
Some of these test values are not valid.  The params field isn't used for all types of sensors, so some of the tuples should only have 3 items.  For floats, params are optional, but we need at least 2: min and max.  From [KATCP guidelines](https://github.com/ska-sa/katcp-python/blob/master/doc/NRF-KAT7-6.0-IFCE-002-Rev5-1.pdf):

![image](https://user-images.githubusercontent.com/20420754/81286195-f59a1f00-9060-11ea-8182-f1ab8c901321.png)

```suggestion
            'a-discrete': ('A discrete sensor', '', 'discrete', 'a', 'b', 'c'),
            'a-timestamp': ('A timestamp sensor', '', 'timestamp'),
            'a-float': ('A float sensor', '', 'float', 1234.42, 5678.99),
            'a-boolean': ('A boolean sensor', '', 'boolean'),
            'an-address': ('An address sensor', '', 'address'),
            'a-lru': ('A LRU sensor', '', 'lru'),
```